app/views/recipes/shopping_list.html.erb

```diff
@@ -0,0 +1,25 @@
+<br>
+ <h2 class="text-center">Shopping List</h2>
+ <section>
+   <div class="d-flex justify-content-between align-items-center">
+     <div>
+       <p>Amount of food to buy: <span><%=%></span> </p>
+       <p>Total value of food needed:<span><%%></span> </p>
+     </div>
+     <div>
+       <p>Recipe: <%= link_to @recipe.name, recipe_path(@recipe) %></p>
+       <p>Inventory:</p>
+     </div>
+   </div>
+   <table class="table table-striped table-bordered">
+     <thead>
+       <tr>
+         <th scope="col">Food</th>
+         <th scope="col">Quantity</th>
```

Comment:
- [ ] Nice work with the interface 👍 . However, the shopping list does not get rendered properly as you can see below. One of the causes is likely your pat route. Kindly debug and fix this to complement your brillant work so far 👍 

### View:
![image](https://github.com/ritikarawat220/Recipe-App/assets/50990925/fbdb4b60-803b-43e5-bf11-3c6b36c309b8)